### DISCUSSION TITLE
i want to help here, please im bored

### DISCUSSION THREAD

**Discussion by sgrossiP** [blakeblackshear/frigate#7426] [General]
hey, im starting to use frigate so my understanding is quite low but i wanna help develop frigate
anything you dont wanna do that i can do for you?

**Comment by NickM-27** [comment]
[The pinned open issues](https://github.com/blakeblackshear/frigate/issues?q=is%3Aopen+is%3Aissue+label%3Apinned) is a good place to start for things to work on that have been accepted for frigate. Any questions on preferred implementation feel free to create a discussion to ask. 

It's important to create a discussion outlining your proposed implementation so you don't waste time working on something that doesn't fit with frigate or would require major modifications.
